README.md

```diff
@@ -4,3 +4,22 @@ AzureIntegration
 Features that integrate ASP.NET Core with Azure.
 
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+
+
+SiteExtensions
+===
+
+To install a nightly preview of the ASP.NET Core runtime site extension for testing purposes:
+1. In the Azure portal select App Services -> your site -> Application settings
+1. Set `SCM_SITEEXTENSIONS_FEED_URL` application setting to `https://dotnet.myget.org/F/aspnetcore-release/`
+1. Go to `Advanced Tools` -> `Site extensions` -> `Gallery`
```

Comment:
Might help to start at the `DEVELOPMENT TOOLS` heading i.e. `DEVELOPMENT TOOLS` -> `Advanced Tools`&hellip;